assets/scss/_custom.scss

```
@@ -639,3 +639,10 @@ body.td-documentation {
   }
 }
 
+
+// Remove shadow from code in table which is added by Docsy
+.td-content {
+  p code, li > code, table code {
```

```suggestion
   table code {
```
I think we just want this within tables. Alternatively, we just want this for `<code>` elements in _tool-reference_ pages.

---

I agree with this suggestion.
Can we get these changes in without the CSS updates and pursue the style changes after merging?
The "code" text is a bit small.

---

Text small for "code" is due to Docsy style customization which can be tuned in the same way, maybe latter?